code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -229,15 +235,18 @@
 	new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
 
 /obj/item/storage/box/syndie_kit/emp
-	name = "boxed EMP kit"
+	name = "box"
 
 /obj/item/storage/box/syndie_kit/emp/PopulateContents()
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/grenade/empgrenade(src)
-	new /obj/item/implanter/emp(src)
+	for(var/i in 1 to 5)
+		new /obj/item/grenade/empgrenade(src)
+
+/obj/item/storage/box/syndie_kit/flashbang
+	name = "box"
+
+/obj/item/storage/box/syndie_kit/flashbag/PopulateContents()
+	for(var/i in 1 to 4)
+		new /obj/item/grenade/flashbang(src)
```

Comment:
Traitors tend to be hurt by flashbangs more than aided. Useless addition.

Reply:
Not if you game the protections, and there are plenty of those protections in the uplink. Syndicate spacesuit helmet alone is good enough for this, as is the laser tag helmet. They're fairly solid for use in fighting non-sec over sec and borgs without completely compromising the room, like an emp does.